Duplicati/Server/webroot/ngax/scripts/services/EditUriBuiltins.js

```diff
@@ -339,10 +339,10 @@ backupApp.service('EditUriBuiltins', function (AppService, AppUtils, SystemInfo,
     ];
 
     EditUriBackendConfig.parsers['s3'] = function (scope, module, server, port, path, options) {
-        if (options['--aws_access_key_id'])
-            scope.Username = options['--aws_access_key_id'];
-        if (options['--aws_secret_access_key'])
-            scope.Password = options['--aws_secret_access_key'];
+        if (options['--aws-access-key-id'])
```

Comment:
Does anyone have an S3 backend that they can test with?  I'm not sure if `options` here is guaranteed to contain the new name, or if it's possible to contain the old deprecated name.

/cc @kenkendk, @drwtsn32x, @ts678